projects/browser-phase-1.md

```diff
@@ -0,0 +1,155 @@
+# Browser Instrumentation (Phase 1) Proposal
+
+## Background and Description
+
+OpenTelemetry is rebooting telemetry for the Browser as a sequence of small, highly focused projects. The first project will focus on creating instrumentation for the browser runtime and a small set of browser libraries, plus the prerequisite API and data modeling work needed to create this instrumentation.
+
+## Current Challenges
+
+OpenTelemetry currently has a NodeJS-focused javascript implementation that is also capable of running in the browser. However, the requirements for a successful browser observability solution differ enough from NodeJS that a specialized solution is required.
+
+* Loading and unloading in the browser are specialized tasks that are very different from booting and shutting down a NodeJS application. Unloading in particular is under strict time and resource constraints.
+* Memory, cpu, and networking resources are severely constrained, especially on mobile devices.
+* A lack of compression and gRPC in browsers
```

Comment:
```suggestion
* A lack of compression and gRPC in browsers
* Sending and offloading telemetry data requires additional considerations, since browser windows can be closed at any time
```

Maybe worth to call this out additionally as well? Or is [beacon](https://caniuse.com/beacon) good enough these days so there is nothing to worry about anymore?

Reply:
This is listed above under "Loading and unloading"

Reply:
ah, then I missed that, thanks!